packages/react/src/components/OverflowMenuV2/OverFlowMenuV2-test.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright IBM Corp. 2016, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { act, fireEvent, render, screen } from '@testing-library/react';
```

Comment:
fireEvent can be replaced by userEvent
import userEvent from '@testing-library/user-event';

Reply:
Done